.github/actions/start-opensearch/action.yml

```diff
@@ -0,0 +1,128 @@
+name: 'Start OpenSearch'
+description: 'Downloads latest build of OpenSearch, installs a plugin, executes and script and then starts OpenSearch on localhost:9200'
+
+inputs:
+  platform:
```

Comment:
Can we drop this parameter in favor of checks from the environment variables like `${{ runner.os != 'Windows'}}`?

> The operating system of the runner executing the job. Possible values are `Linux`, `Windows`, or `macOS`.

_From https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables_